bin/ops.bash

```diff
@@ -17,6 +17,21 @@ usage() {
   exit 1
 }
 
+gatekeeper_cleanup() {
+  local kubeconfig="$1" ns="${2:-gatekeeper-system}" rel="${3:-gatekeeper-templates}"
+
+  with_kubeconfig "${kubeconfig}" bash -cu '
+    echo "Running Gatekeeper Cleanup Resources"
```

Comment:
Log instead of echo and no need to do it inside the new shell.

```suggestion
  log_info "Running Gatekeeper Cleanup Resources"
  
  with_kubeconfig "${kubeconfig}" bash -cu '
```